ci-operator/config/openshift/os/openshift-os-release-4.14.yaml

```
@@ -2,7 +2,7 @@ base_images:
   coreos_coreos-assembler_latest:
```

```suggestion
  coreos_coreos-assembler_4.14:
```

In the referenced PR: https://github.com/openshift/release/pull/33905, this line was changed to indicate the version. Want to do that here too?

---

I missed that change!  Thanks @marmijo